src/services/codefixes/inferFromUsage.ts

```diff
@@ -221,49 +216,41 @@ namespace ts.codefix {
         }
     }
 
-    function getTypeAccessiblityWriter(checker: TypeChecker): EmitTextWriter {
-        let str = "";
+    function isTypeAccessible(type: Type, enclosingDeclaration: Declaration, checker: TypeChecker): boolean {

```

Comment:
@weswigham Is there a better way to do this now?

Reply:
`typeToTypeNode` itself should have an argument that lets you pass in visibility tracking hooks in the same way without needing to print the type.

Reply:
Had to add that parameter as an internal overload -- should it be public?

Reply:
I don't _think_ we want to encourage using it? `EmitTextWriter` is internal, so it's not like the other method was public.

Reply:
OK, will leave it internal.